CONTRIBUTING.md

```diff
@@ -217,12 +217,12 @@ fix: replace the word "deploy" with "publish" everywhere.
 We should be consistent with the word that describes how we get a worker to the edge. The command is `publish`, so let's use that everywhere.
 ```
 
-### Types of changes (and deviation from semver)
+### Types of changes
 
 We use the following guidelines to determine the kind of change for a PR:
 
-- Bugfixes and new features are considered to be 'patch' changes. If the new feature is experimental and its behaviour may functionally change, be sure to log warnings whenever they're used. (You'll note that this is where we deviate from semver, which otherwise suggests that behaviour/api changes should go into minor releases. We may revisit this in the future.)
-- New deprecation warnings for future breaking changes are considered as 'minor' changes. These changes shouldn't break existing code, but the deprecation warnings should suggest alternate solutions to not trigger the warning.
+- Bugfixes are considered to be 'patch' changes.
+- New features (both experimental and stable) and new deprecation warnings for future breaking changes are considered as 'minor' changes. These changes shouldn't break existing code, but the deprecation warnings should suggest alternate solutions to not trigger the warning. If the new feature is experimental and its behaviour may functionally change, be sure to log warnings whenever they're used.
```

Comment:
I don't necessarily think feature-flagged/toggleable experimental changes should be a minor version bump—we add a _lot_ of experimental features, and this'll mean that every experimental features require two bumps; one when it's made experimental, and one when it's made stable.

Reply:
I was discussing this with @JacobMGEvans and I agree with you @penalosa tbh

Reply:
I've added it to the agenda for this afternoon

Reply:
I could be convinced otherwise. Look forward to the discussion.